subject.py

```diff
@@ -0,0 +1,178 @@
+import datajoint as dj
+
+import reference
+
+schema = dj.schema(dj.config['names.%s' % __name__], locals())
+
+
+
+@schema
+class Species(dj.Manual):
+    # <class 'subjects.models.Species'>
+    definition = """
+    species_id:			int             # species id
+    ---
+    binomial:			varchar(255)	# binomial
+    display_name:		varchar(255)	# display name
+    """
+
+
+@schema
+class Strain(dj.Manual):
+    # <class 'subjects.models.Strain'>
+    definition = """
+    strain_id:			int             # strain id
```

Comment:
Same thing here - unless they mean to use some unique but descriptive shorthand as the `strain_id` (probably not the case given it's an `int`), it would be better to have `descriptive_name` be the primary key. I'd rename it to `strain_name` or something along that line.

Reply:
yep. i think for some of these id types I just copied in on the side of caution since it doesn't pollute so much w/r/t classes.. but in any event, will do